File: Modul6/PRAK601-2310817110008-Muhammad_Raihan.c
#include <stdio.h>

void main(){
    int baris,kolom;
    scanf("%d %d", &baris, &kolom);
    int matriks[baris][kolom];

    for(int i=0; i<baris; i++){
        for(int j=0; j<kolom; j++){
            scanf("%d", &matriks[j][i]);
        }
    }
    printf("\n");
    for(int i=0; i<baris; i++){
        for(int j=0; j<kolom; j++){
            printf("%d ", matriks[j][i]);
        }
        printf("\n");
    }
}

File: Modul6/PRAK601-2310817110008-Muhammad_Raihan.py
matriks = []
baris,kolom = map(int, input().split())
data = list(map(int, input().split()))
for i in range(0,baris,1):
    elemen = []
    for j in range(0,kolom,1):
        elemen.append(data[j+(i*kolom)])
    matriks.append(elemen)

print("")
for i in range(0,baris,1):
    for elemen in matriks[i]:
        print(elemen,end=' ')
    print("")